src/instructlab/configuration.py

```diff
@@ -206,6 +206,7 @@ class _evaluate(BaseModel):
     base_model: str
     branch: Optional[str] = None
     base_branch: Optional[str] = None
+    gpus: int = 1
```

Comment:
Can we make this Optional?

I think it be good if we made that the standard practice, because otherwise, every time a new required parameter gets added, previously generated `config.yaml` is broken and causes errors at `ilab` startup.

In a perfect world, we'd go back and do that for everything added since the last CLI release, though that's beyond the scope of this PR

Reply:
I should clarify a bit that I'm guessing the only change needed to get he behavior I'm suggesting is to mark it optional ... i haven't tried it yet

Reply:
Good catch.  It should be optional in the config.